packages/storybook-ui-components/package.json

```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/cockroachdb/ui#readme",
   "dependencies": {
     "@cockroachlabs/icons": "^0.4.0",
-    "@cockroachlabs/ui-components": "next"
+    "@cockroachlabs/ui-components": "link:packages/ui-components"
```

Comment:
I like the look of this, but what happens during deploy? It just uses what's local to the repo? Which I guess is fine since it simply builds a static site.

Reply:
> but what happens during deploy? 
good question! It will use local repo, but it might also require to build `ui-components` package as an extra step.

Reply:
@nathanstilwell , I've updated storybook publish configuration to build packages because local repository doesn't contain bundled files.